dev-libs/libdivsufsort/libdivsufsort-2.0.1.ebuild

```diff
@@ -1,20 +1,27 @@
-# Copyright 1999-2012 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=4
+EAPI=6
 inherit cmake-utils multilib
 
+if [[ ${PV} != 9999* ]]; then
```

Comment:
Try it on the live ebuild and see that it works as desired. But feel free to drop that crap altogether, and just have split code for -2.0.1 and -9999. Those universal ebuilds only increase maintenance burden in the end.